tornado/test/locks_test.py

```diff
@@ -1,9 +1,21 @@
-from __future__ import absolute_import, division, print_function, with_statement
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
 
 from datetime import timedelta
+import unittest
```

Comment:
Files in `tornado/test/` should always import unittest with `from tornado.test.util import unittest`. It doesn't seem to have hurt here but the interaction between unittest and unittest2 has been fragile in the past.